content/django-vs-flask.md

```diff
@@ -0,0 +1,73 @@
+# Django vs Flask: which one should you choose for your project?
+Django and Flask are among the most popular Python web frameworks.
+In this article we're going to review their main strengths and weaknesses to help you decide which one is the better fit for your project.
+
+
+# Django
+
+## Approach
+Django opts for an all batteries included approach.
+This means that we have everything we need to build our project included into the framework.
+from a powerful ORM to an authentication system or a templating engine.
+Django takes care of it all without the need to learn any extra libraries.
+
+It also has the concept of pluggable apps, that means that you can drop an app folder inside your project and get its functionality right away
+with very little configuration.
+
+## Main features
+- Django has a very powerful ORM that exposes a lot of SQL to the python programming language.
+- Authentication and Authorization built in, without any extra configuration.
+- A built in admin site.
+- A built in mailing system.
+
+## Pitfalls
+- Tightly coupled with its toolset, Django is sort of difficult to use in combination with other tools (replacing the form library for example).
+- The default scaffolding, while not bad for small projects, does not scale without modification when trying to manage things such as multiple environments.
+- It implements the Model View Template pattern, and requires some effort to implement other architecture types with it.
+
+
+# Flask
+
+## Approach
+Born as a combination of an http server with a template engine zipped together, Flask opts for a very minimalistic approach.
+It provides some minimal structure, but leaves the decisions up to you.
+This makes flask very easy to extend with practically whatever you want.
+
+## Main features
+- Very lightweight.
+- It requires little to no configuration.
+- Not opinionated, you have total freedom to decide the project structure, from a single file to an elaborate folder system.
+- When you need some structure, you have blueprints to structure your app.
+- Easy to integrate with other python libraries.
+
+## Pitfalls
+- Since it's very lightweigth, while the framework itself doesn't need any special configuration, the used libraries often do require config.
```

Comment:
```suggestion
- Since it's very lightweight, while the framework itself doesn't need any special configuration, the used libraries often do require config.
```